business/authentication/openid_auth_controller.go

```diff
@@ -372,6 +372,7 @@ func (c OpenIdAuthController) redirectToAuthServerHandler(w http.ResponseWriter,
 	nonceCookie := http.Cookie{
 		Expires:  expirationTime,
 		HttpOnly: true,
+		Secure:   conf.IsServerHttps(),
```

Comment:
Yes, I almost think it has to be because the "Secure" flag is just that -- a flag in the cookie. So the set-cookie header doesn't have "Secure=true" or "Secure=false" its just "Secure" or not specified.

And yes the default is false.

e.g. https://http.dev/set-cookie#secure